test/libsolidity/semanticTests/builtins/balance_other_contract.sol

```diff
@@ -0,0 +1,29 @@
+contract Other {
```

Comment:
I'd rename this directory to `isoltestTesting` or something else, but `builtins` here suggests precompiles or some solidity builtins.

Reply:
Done.